src/widgets/ThemedWrappers.h

```diff
@@ -0,0 +1,114 @@
+/*!********************************************************************
+
+   Audacity: A Digital Audio Editor
+
+   @file ThemedWrappers.h
+
+   @author Vitaly Sverchinsky
+
+   A set of classes which are simplify theme support
```

Comment:
"which simplify"

Reply:
Perhaps say "which decorate given base classes with simplified theme support"

This is sort of a "decorator pattern" only it inserts behavior with inheritance, not object composition.